include/ts/apidefs.h.in

```diff
@@ -51,11 +51,62 @@
 #define tsapi
 #endif
 
+/** Apply printf format string compile-time argument checking to a function.
+ *
+ *
+ * For example, given the following function from DiagsTypes.h:
+ *
+ *   class Diags {
```

Comment:
See [`@code`](https://www.doxygen.nl/manual/commands.html#cmdcode), [`@endcode`](https://www.doxygen.nl/manual/commands.html#cmdendcode). [Example](https://github.com/apache/trafficserver/blob/master/include/ts/ts.h#L104).

Reply:
Nice. Thank you.